ncsa-teststand/MT/docker-compose.yaml

```diff
@@ -7,18 +7,15 @@ services:
     container_name: mtm1m3-sim
     environment:
       - LSST_DDS_PARTITION_PREFIX=${SITE_PARTITION_PREFIX}
-      - LSST_DDS_RESPONSIVENESS_TIMEOUT=15s
```

Comment:
Same as before, are you sure you want to remove this?

Reply:
Ditto from above.